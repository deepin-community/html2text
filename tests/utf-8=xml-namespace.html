<HTML><HEAD>
<META http-equiv="Content-Type" content="text/html; charset=utf-8">
</HEAD>
<BODY>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black'>+7 
(931) 009-16-22</SPAN></B><B style="mso-bidi-font-weight: normal"><SPAN 
lang=EN-US 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; mso-ansi-language: EN-US'><?xml:namespace 
prefix = "o" ns = "urn:schemas-microsoft-com:office:office" 
/><o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>(звонки или 
WhatsApp)<B style="mso-bidi-font-weight: normal"><SPAN 
style="COLOR: red"><o:p></o:p></SPAN></B></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN lang=EN-US 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; mso-ansi-language: EN-US'><A><SPAN 
style="COLOR: blue">Magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">-</SPAN><SPAN 
style="COLOR: blue">magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">2020@mail.ru</SPAN></A></SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue'> 
<o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Производитель 
одноразовых медицинских масок, защитных костюмов, халатов и антисептиков 
предлагает <B style="mso-bidi-font-weight: normal"><U>средства индивидуальной 
защиты для предприятий</U></B> по оптовым ценам.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: red'>Наши цены 
вас приятно удивят:<o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'>1. 
<SPAN style="COLOR: blue">3-слойные шитые маски</SPAN></SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 40 гр. 3 слоя) - от 13 рублей. <o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>2.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Антисептики собственного производства</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(рецепт ВОЗ, спирт не менее 70%, глицерин, перекись водорода, вода) - от 500 
рублей/литр.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>3.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Антивирусные защитные костюмы "Каспер"</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 25/40/50/60 гр.) - от 155 рублей.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>4.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Защитные костюмы "Тайвек"</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(ламинированный спанбонд 40/60/80 гр.) - от 300 рублей. <o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>5.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Медицинские халаты</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 25/40/50/60 гр.) - от 85 рублей.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>6.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Высокие защитные бахилы <?xml:namespace prefix = "st1" ns = 
"urn:schemas-microsoft-com:office:smarttags" /><st1:metricconverter w:st="on" 
ProductID="40 см">40 см</st1:metricconverter>.</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 25/40/50/60 гр.) - от 30 рублей. <o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Также у нас есть 
медицинские маски Китайского производства, шапочки, нарукавники, фартуки<B 
style="mso-bidi-font-weight: normal"> <SPAN style="COLOR: red">и любые другие 
средства защиты</SPAN>.<o:p></o:p></B></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue'>Мы 
являемся производителем</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>, поэтому наши цены 
ниже конкурентов.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Вся продукция 
сертифицирована.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Возможна доставка по 
договоренности в любой регион.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Для постоянных 
партнеров и на крупные объемы цены еще ниже.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Работаем за 
наличный/безналичный расчет с НДС.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: red'>Товар от 
производителя.<o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Производство и склад 
в г. Новосибирск. Образцы товаров в наличии в Москве.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 15pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 16pt; FONT-FAMILY: "Arial","sans-serif"'>Позвоните нам или 
напишите в WhatsApp:</SPAN></B><SPAN 
style='FONT-SIZE: 16pt; FONT-FAMILY: "Arial","sans-serif"'> <BR></SPAN><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black'>+7 
(931) 009-16-22</SPAN></B><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"'><o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>или напишите на 
почту: </SPAN><SPAN lang=EN-US 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; mso-ansi-language: EN-US'><A><SPAN 
style="COLOR: blue">Magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">-</SPAN><SPAN 
style="COLOR: blue">magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">2020@mail.ru</SPAN></A></SPAN><SPAN 
lang=EN-US style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'> 
</SPAN><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p></o:p></SPAN></DIV></BODY></HTML>
