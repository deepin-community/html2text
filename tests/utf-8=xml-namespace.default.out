                              ++77 ((993311)) 000099--1166--2222
                             (звонки или WhatsApp)
                           MMaaggnnuumm--mmaaggnnuumm22002200@@mmaaiill..rruu
 
Производитель одноразовых медицинских масок, защитных костюмов, халатов и
антисептиков предлагает _?с_?р_?е_?д_?с_?т_?в_?а_ _?и_?н_?д_?и_?в_?и_?д_?у_?а_?л_?ь_?н_?о_?й_ _?з_?а_?щ_?и_?т_?ы_ _?д_?л_?я_ _?п_?р_?е_?д_?п_?р_?и_?я_?т_?и_?й по
оптовым ценам.
 
?Н?а?ш?и ?ц?е?н?ы ?в?а?с ?п?р?и?я?т?н?о ?у?д?и?в?я?т::
 
11.. 33--?с?л?о?й?н?ы?е ?ш?и?т?ы?е ?м?а?с?к?и (спанбонд 40 гр. 3 слоя) - от 13 рублей.
22.. ?А?н?т?и?с?е?п?т?и?к?и ?с?о?б?с?т?в?е?н?н?о?г?о ?п?р?о?и?з?в?о?д?с?т?в?а (рецепт ВОЗ, спирт не менее 70%,
глицерин, перекись водорода, вода) - от 500 рублей/литр.
33.. ?А?н?т?и?в?и?р?у?с?н?ы?е ?з?а?щ?и?т?н?ы?е ?к?о?с?т?ю?м?ы ""?К?а?с?п?е?р"" (спанбонд 25/40/50/60 гр.) - от 155
рублей.
44.. ?З?а?щ?и?т?н?ы?е ?к?о?с?т?ю?м?ы ""?Т?а?й?в?е?к"" (ламинированный спанбонд 40/60/80 гр.) - от 300
рублей.
55.. ?М?е?д?и?ц?и?н?с?к?и?е ?х?а?л?а?т?ы (спанбонд 25/40/50/60 гр.) - от 85 рублей.
66.. ?В?ы?с?о?к?и?е ?з?а?щ?и?т?н?ы?е ?б?а?х?и?л?ы 4400 ?с?м.. (спанбонд 25/40/50/60 гр.) - от 30 рублей.
 
     Также у нас есть медицинские маски Китайского производства, шапочки,
             нарукавники, фартуки ?и ?л?ю?б?ы?е ?д?р?у?г?и?е ?с?р?е?д?с?т?в?а ?з?а?щ?и?т?ы..
                                        
        ?М?ы ?я?в?л?я?е?м?с?я ?п?р?о?и?з?в?о?д?и?т?е?л?е?м, поэтому наши цены ниже конкурентов.
                                        
                        Вся продукция сертифицирована.
                                        
              Возможна доставка по договоренности в любой регион.
                                        
          Для постоянных партнеров и на крупные объемы цены еще ниже.
                                        
                Работаем за наличный/безналичный расчет с НДС.
                                        
                            ?Т?о?в?а?р ?о?т ?п?р?о?и?з?в?о?д?и?т?е?л?я..
                                        
  Производство и склад в г. Новосибирск. Образцы товаров в наличии в Москве.
                                        
 
                    ?П?о?з?в?о?н?и?т?е ?н?а?м ?и?л?и ?н?а?п?и?ш?и?т?е ?в WWhhaattssAApppp::
                              ++77 ((993311)) 000099--1166--2222
               или напишите на почту: Magnum-magnum2020@mail.ru
